services/ui-src/src/routes/Intro/Intro.tsx

```diff
@@ -1,29 +1,138 @@
-  const { intro } = verbiage;
+  const { intro, body, pageLink } = verbiage;
+  const mqClasses = makeMediaQueryClasses();
+  const navigate = useNavigate();
+
+  const [section1, section2, section3] = body.sections;
+
   return (
-    <ReportPage data-testid="intro-view">
+    <BasicPage sx={sx.layout} data-testid="intro-view">
       <Box sx={sx.leadTextBox}>
         <Heading as="h1" sx={sx.headerText}>
           {intro.header}
         </Heading>
-        <Text>{intro.body}</Text>
       </Box>
-    </ReportPage>
+      <div>
+        <IntroSection index={1} content={section1} key={`section-1`}>
+          <Flex sx={sx.widgetsContainer} className={mqClasses}>
+            <Box sx={sx.imagelessWidgetContainer}>
+              <Text sx={sx.imagelessWidgetTitle}>{section1.widget?.title}</Text>
+              <Box>
+                {section1.widget?.descriptionList.map((description, index) => (
+                  <Text key={index}>{description}</Text>
+                ))}
+              </Box>
+            </Box>
+          </Flex>
+        </IntroSection>
+        <IntroSection index={2} content={section2} key={`section-2`}>
+          <Flex sx={sx.widgetsContainer} className={mqClasses}>
+            <Box>
+              <Image
+                src={section2.img?.src}
+                alt={section2.img?.alt}
+                sx={sx.image}
+              />
+              <Text sx={sx.additionalInfo}>{section2.img?.description}</Text>
+            </Box>
+            <SpreadsheetWidget content={section2.spreadsheetWidget!} />
+          </Flex>
+        </IntroSection>
+        <IntroSection index={3} content={section3} key={`section-3`}>
+          <Flex sx={sx.widgetsContainer} className={mqClasses}>
+            <Box>
+              <Image
+                src={section3.img?.src}
+                alt={section3.img?.alt}
+                sx={sx.image}
+              />
+            </Box>
+          </Flex>
+        </IntroSection>
+      </div>
+      <Box sx={sx.pageLinkContainer}>
+        <Button
+          className={mqClasses}
+          sx={sx.pageLink}
+          onClick={() => navigate(pageLink.route)}
+          rightIcon={<Image src={nextIcon} alt="Link Icon" height="1rem" />}
+        >
+          {pageLink.text}
+        </Button>
+      </Box>
+    </BasicPage>
   );
 };
 
 const sx = {
+  layout: {
+    ".contentFlex": {
+      marginTop: "3.5rem",
+    },
+  },
   leadTextBox: {
+    width: "100%",
     marginBottom: "2.25rem",
   },
   headerText: {
     marginBottom: "1rem",
     fontSize: "2rem",
     fontWeight: "normal",
   },
+  widgetsContainer: {
+    marginTop: "1rem",
+    gridGap: "2rem",
+    flexDirection: "column",
+    "&.desktop": {
+      flexDirection: "row",
+    },
+  },
+  imagelessWidgetContainer: {
+    marginTop: "1rem",
+    paddingLeft: "1rem",
+    borderLeft: ".3rem solid",
+    borderColor: "palette.main",
+  },
+  imagelessWidgetTitle: {
+    fontWeight: "bold",
+  },
```

Comment:
Since these aren't really widgets anymore, we should change these style names to something else. Maybe: `sectionContentContainer`, `sectionContentContainerTitle`, etc.

Reply:
Can rename widgetsContainer to sectionContent but i'm not sure about the imagelessWidgetContainer->sectionContentContainer. If we're keeping the Spreadsheet Widget name and this small part looks exactly like it without the image and a change in bold title/description then I'm not sure what else to call it

Reply:
Ah gotcha. How about `spreadsheetWidgetContainer` and `spreadsheetWidgetTitle`?

Reply:
I think I see the confusion here. So theres 2 "widgets", for lack of a better term, on the page. We have the top one which was using the imagelessWidgetContainer and imagelessWidgetTitle class: 

![Screen Shot 2022-08-03 at 6 08 54 PM](https://user-images.githubusercontent.com/19439679/182721126-7d7aefd6-e574-423f-a421-4049dd45b9b1.png)

and we have the bottom one that uses its own reusable component with its own classes:
![Screen Shot 2022-08-03 at 6 09 53 PM](https://user-images.githubusercontent.com/19439679/182721243-ecf55d66-1a9c-4837-9914-288c630c4e20.png)

I wouldn't want to rename the imagelessWidgetContainer or imagelessWidgetTitle css class to spreadsheetWidgetContainer or sectionContentContainer since that seems either not what it is or too vague.

I've settled, for now, on reducing the class names to widgetContainer and widgetTitle to reflect this but happy to change it to something that could be more fitting.